src/main/resources/schema/mutation.graphqls

```diff
@@ -1,5 +1,26 @@
 type Mutation {
-  createWish(input: WishInput!): Wish!
-  deleteWish(id: ID!): Boolean!
-  updateWish(id: ID!, input: WishInput!): Wish
+  "Create a new App Wish "
+  addWish(input: WishInput): Wish
+
+  "Edit an AppWish, providing the ID and the fields to update"
+  editWish(id: ID!, input: WishInput): Wish
+
+  "Delete an AppWish by ID"
+  deleteWish(id: ID!): Boolean
+
+  "Add a new comment... The userID is the one who created this comment"
+  addComment(targetID:ID!, userID: ID!, msg: String!): Comment
```

Comment:
I think we don't need user ID in inputs - it will probably be figured out from authorization context.

Reply:
We may also think of creating input types for mutations

Reply:
Will we pass the UserID through headers in our requests?

Reply:
Done :smile: